### PR TITLE
FSD architecture

### PR DESCRIPTION
**Pull request by zinuu4**

### CODE REVIEW

**Review by NurkenInc — CHANGES_REQUESTED** [review]
Great job for first time migrating to FSD! ofc there are a lot of work that should be done

**Review by almalmalm — APPROVED** [review]
Good

File: src/widgets/header/ui/search-btn/index.ts
[the remark this answers is not in the document]
Comment (zinuu4) [reply src/widgets/header/ui/search-btn/index.ts:1]: In this file locates only trigger button for search modal

**Review by NurkenInc — APPROVED** [review]
Well done!

**Pull request merged**